src/App.tsx

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
-import logo from './logo.svg';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Home from './pages/home';
+import Results from './pages/results';
+import Details from './pages/details';
+import About from './pages/about'
+import Navigation from './component/Navbar';
```

Comment:
Naming of components needs to match conventions:

-  PascalCase component names
- Export component matching name: Navbar exports <Navbar>, etc